### PR TITLE
Update appsearch.asciidoc

### PR DESCRIPTION
**Pull request by InbarShimshon**
AppSearch starting 7.7.0 was dubbed to Enterprise Search in Cloud that contains both WorkPlace and AppSearch.
From my testing when connecting AppSearch to   https://19bb0ad019e54a4c897957779b7ca900.ent-search.eu-central-1.aws.cloud.es.io/api/ent/v1/internal/stats the output is not one supported in https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-metricset-appsearch-stats.html so we should be more explicit in this not being supported for now in cloud where AppSearch falls under the Enterprise Search umbrella.

## What does this PR do?

Clarifies that this module is not supporting Cloud AppSearch Metricbeat monitoring. 

## Why is it important?

Tickets get in support - its a confusing (beta) topic so the clearer we are in the docs the less time it will be for support to test, reproduce open SDHs and take time from Dev to only find out it is not supported.

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: metricbeat/docs/modules/appsearch.asciidoc [hunk 1]
```diff
@@ -40,5 +40,7 @@ The following metricsets are available:
 
 * <<metricbeat-metricset-appsearch-stats,stats>>
 
-include::appsearch/stats.asciidoc[]
+include::appsearch/stats.asciidoc[] 
+
+Note: This module does not support AppSearch within Cloud.
```
Comment (EamonnTP) [inline metricbeat/docs/modules/appsearch.asciidoc:45]:
```suggestion
Note: This module does not support App Search within {ecloud}.
```

**Comment by EamonnTP** [comment]
jenkins run the tests please

**Comment by EamonnTP** [comment]
Hi @InbarShimshon thanks for your contribution! Apologies, I should have picked this up on my first review. It looks like you have updated the generated file in the `7.10` branch, rather than the source file in the `master` branch.

You should add the note to the bottom of the source file here: https://github.com/elastic/beats/blob/master/x-pack/metricbeat/module/appsearch/_meta/docs.asciidoc and then run the update (`make update`) to update the generated file.

You can then back port your changes to the `7.x` and `7.10` branches. 

Thanks again!

**Comment by InbarShimshon** [comment]
@EamonnTP thank you - do I need to do anything else now? I dont think it allowed em to update the master btw - the UI of GH for managing docs is very messy is my humble in put. Thank you for helping me on this one!

**Comment by EamonnTP** [comment]
Hi @InbarShimshon It might be easier to close this PR and open a new PR against the `master` branch rather than the `7.10` branch. On the `master` branch, update the [source file](https://github.com/elastic/beats/blob/master/x-pack/metricbeat/module/appsearch/_meta/docs.asciidoc) and then run the update (`make update`) to update the generated file.

If need be, I can then help with back porting the changes to the `7.x` and `7.10` branches.

Thanks!

**Comment by EamonnTP** [comment]
This update has been addressed in https://github.com/elastic/beats/pull/23935

**Pull request closed without merging**